### ISSUE TITLE
schema-reader Error: Error: config/permission.php not loaded. Run [php artisan config:clear] and try again.

### ISSUE THREAD

**Issue by kinzinho666** [VemtoOrg/vemto2-issues#42]
**Error Message:**
schema-reader Error: Error: config/permission.php not loaded. Run [php artisan config:clear] and try again.

**Stack:**
```
schema-reader Error: #0 [internal function]: CreatePermissionTables->up()
#1 phar://C:/Users/k1/AppData/Local/Programs/vemto/resources/static/VMTTL1/classes/MigrationDecoder.php(22): ReflectionMethod->invoke(Object(CreatePermissionTables))
#2 phar://C:/Users/k1/AppData/Local/Programs/vemto/resources/static/VMTTL1/index.php(77): MigrationDecoder->decode()
#3 phar://C:/Users/k1/AppData/Local/Programs/vemto/resources/static/VMTTL1/common/Vemto.php(107): {closure}()
#4 phar://C:/Users/k1/AppData/Local/Programs/vemto/resources/static/VMTTL1/index.php(18): Vemto::execute('schema-reader', Object(Closure))
#5 C:\Users\k1\AppData\Local\Programs\vemto\resources\static\VMTTL1(14): require('phar://C:/Users...')
#6 {main}
```

---

Vemto 2.0.5
win32 10.0.22631
Locale: pt-BR
App version: 2.0.5

**Comment by TiagoSilvaPereira** [comment]
Hi @kinzinho666 thanks for reporting. Have you tried running the command ```php artisan config:clear```?

**Comment by kinzinho666** [comment]
Yes i try, same problem,  i try use another tabel structure but have more than 15 tabels, so need license .

**Comment by TiagoSilvaPereira** [comment]
In this case, one of your migrations probably has logic that Schema Reader cannot interpret. I'm currently rewriting Schema Reader to solve this kind of problem. By next week, we will have a new version to solve that. For now, I recommend testing with a new, blank Laravel project.

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7
